config.yaml

```diff
@@ -0,0 +1,46 @@
+#cloud-config
```

Comment:
This sample is now part of the test in kairos: https://github.com/kairos-io/kairos/pull/2291/files#diff-1ff1699e612ac7f8c508e5f9f6a784b37441b01b8cfdebd8da3b068280385247R87

I'll remove it from here.